desertbot/desertbot.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, factory: 'DesertBotFactory', config: Config):
             'finished': []
         }
 
+        if 'sasl_username' in self.config:
```

Comment:
This should just be an "if self.config.getWithDefault('sasl_username', None) is not None" check, in case sasl_username is null in the config

Reply:
True. Fixed.